pkgs/servers/nextcloud/notify_push.nix

```diff
@@ -20,6 +26,20 @@ rustPlatform.buildRustPackage rec {
   cargoHash = "sha256-bO3KN+ynxNdbnFv1ZHJSSPWd4SxWQGIis3O3Gfba8jw=";
 
   passthru = rec {
+    app = fetchNextcloudApp {
+      appName = "notify_push";
+      appVersion = version;
+      hash = "sha256-4yCs4Q25PhYVICAIFlNiRTOFvL0JdmUwR5bNxp54GiA=";
+      license = "agpl3Plus";
+      homepage = "https://github.com/nextcloud/notify_push";
+      url = "https://github.com/nextcloud-releases/notify_push/releases/download/v${version}/notify_push-v${version}.tar.gz";
+      description = ''
+        Push update support for desktop app.
+
+        Once the app is installed, the push binary needs to be setup. You can either use the setup wizard with `occ notify_push:setup` or see the [README](http://github.com/nextcloud/notify_push) for detailed setup instructions
```

Comment:
```suggestion
```

Reply:
Added a different sentence about the server-side requirements.